lxd/ovn.go

```diff
@@ -0,0 +1,115 @@
+package main
+
+import (
+	"context"
+	"net"
+	"strings"
+
+	"github.com/canonical/lxd/lxd/state"
+	"github.com/canonical/lxd/shared/api"
+)
+
+type OVNLogMonitor struct {
+	stateFunc func() *state.State
+	conn      net.PacketConn
+}
+
+// NewOVNLogMonitor returns a new OVNLogMonitor.
+func NewOVNLogMonitor(fn func() *state.State) *OVNLogMonitor {
```

Comment:
just a nit but `fn` for a function argument is pretty vague. How about `stateFunc`?